README.md

```diff
@@ -63,11 +64,38 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         with:
-          ssm_parameter: 'my_parameter_name'
+          ssm_parameter_list: 'my_parameter_name'
 ```
 
 Example above will set environmental variable `AWS_SSM_MY_PARAMETER_NAME` with value from the AWS SSM parameter itself.
 
+#### List of parameter values
```

Comment:
Change this into `#### Get multiple parameter values` with description:

`Use comma separated list of strings to fetch multiple parameter values at once:`